framework/auth/forms.py

```diff
@@ -3,6 +3,8 @@
 # TODO: Remove me
 
 from wtforms import ValidationError
+
+from framework import auth
```

Comment:
Don't reverse it or anything, but especially for a PR of this size, it's a good idea not to do irrelevant formatting or line-shuffling changes. I mean, if this fixes something that was broken, cool, but otherwise it's just another potential merge conflict. Best to save those for otherwise tiny PRs if it's important but not urgent.